lua/wikis/commons/Character.lua

```diff
@@ -0,0 +1,64 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Character
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Info = require('Module:Info')
+
+local Character = {}
+
+---@class StandardCharacter
+---@field name string
+---@field pageName string
+---@field releaseDate string?
+---@field iconLight string?
+---@field iconDark string?
+
+local function datapointType()
+	if Info.wikiName == 'dota2' then
+		return 'hero'
+	end
```

Comment:
should we maybe make this a config field in module:info?
(we likely will get additional cases for this, right?)

Reply:
Nah they should be standardized imo going forward, it's just that dota2 has extensive usage of it in non-git tracked moduels, making it PITA to change